mgmt/imgmgr/include/imgmgr/imgmgr.h

```diff
@@ -111,7 +122,10 @@ int imgmgr_state_slot_in_use(int slot);
 int imgmgr_state_set_pending(int slot, int permanent);
 int imgmgr_state_confirm(void);
 int imgmgr_find_best_area_id(void);
-
+int imgmgr_register_callbacks(const imgmgr_dfu_callbacks_t *cb_struct);
+void imgr_dfu_stopped(void);
```

Comment:
was it intention to expose the `img_dfu_foo()`? If so it should have prefix `imgmgr`. If it was not the intention then those functions should be `static`